draft-ietf-acme-acme.md

```diff
@@ -812,9 +812,9 @@ identifier (required, dictionary of string):
   : The identifier itself.
 
 status (required, string):
-: The status of this authorization.  Possible values are: "unknown", "pending",
-"processing", "valid", "invalid" and "revoked".  If this field is missing, then
-the default value is "pending".
+: The status of this authorization.  Possible values are: "pending", "processing",
```

Comment:
I'm in favor of keeping "processing." Boulder doesn't currently implement that status but it should.